metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -136,3 +161,63 @@ export function getNodeAutoInstrumentations(
 
   return instrumentations;
 }
+
+export function getResourceDetectorsFromEnv(): Array<Detector | DetectorSync> {
+  const resourceDetectorsFromEnv =
+    process.env.OTEL_NODE_RESOURCE_DETECTORS?.split(',');
+
+  if (
+    resourceDetectorsFromEnv == null ||
+    resourceDetectorsFromEnv.includes('all')
+  ) {
+    return [
+      // Standard resource detectors.
+      containerDetector,
+      envDetectorSync,
+      hostDetectorSync,
+      osDetectorSync,
+      processDetectorSync,
+      // Cloud resource detectors.
+      alibabaCloudEcsDetector,
+      // Ordered AWS Resource Detectors as per:
+      // https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md#ordering
+      awsEksDetector,
+      awsEc2Detector,
```

Comment:
Any reason you choose just those 2 and not all of them?
like, ecs, lambda etc

Reply:
Added the missing AWS detectors :)